unstructured/ingest/main.py

```diff
@@ -95,18 +102,40 @@ def run(self):
 
 
 @click.command()
+@click.option(
+    "--remote-url",
+    default=None,
+    help="Remote fsspec URL formatted as `protocol://dir/path`, it can contain both "
+    "a directory or a single file.",
```

Comment:
Might as well explicitly mention the supported protocols here, to save the user any confusion.